app/main.py

```diff
@@ -201,6 +216,7 @@ def main():
     server = tornado.httpserver.HTTPServer(app)
     server.bind(int(os.getenv("SDX_GATEWAY_PORT", '8080')))
     server.start(1)
+    bridge = Bridge()
```

Comment:
Does this need to move outside the try : except?

Reply:
I had it before inside the try: except however it got an unresolved ref for the bridge object? i.e. `bridge.stop()`